package.json

```
@@ -1,8 +1,8 @@
 {
-  "name": "developer-documentation",
+  "name": "Developer Documentation",
```

Re-reading the https://docs.npmjs.com/cli/v10/configuring-npm/package-json this is wrong.

```suggestion
  "name": "documentation-developer",
```